WordPress/src/main/java/org/wordpress/android/ui/reader/utils/ReaderImageScanner.java

```diff
@@ -81,6 +77,32 @@ public ReaderImageList getImageList(int minImageWidth) {
     }
 
     /*
+     * returns true if there at least `minImageCount` images in the post content that are at
+     * least `minImageWidth` in size
+     */
+    public boolean hasUsableImageCount(int minImageCount, int minImageWidth) {
```

Comment:
I am just wondering if the common code in this method and `getImageList` can be extracted somehow. I couldn't find a straight forward refactor myself, but thought I'd bring it up anyway. It'd help if we could.